AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -182,27 +189,27 @@ def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size
                     "WARNING: Myelin object not found for axon" +
                     "centroid [y:{0}, x:{1}]".format(y0, x0)
                     )
-
-        stats_array = np.append(stats_array, [stats], axis=0)
+        # Add the stats to the dataframe
+        # Question: should we use float32 as a datatype?
```

Comment:
float64

Reply:
I'd be curious to see how much it impacts performance honestly. Not sure float64 is needed because the uncertainty on the measures probably makes a lot of this added accuracy useless.

Reply:
Fair point. I believe we use float32 currently on master but I wasn't sure if it was really working because of the way the stats_array worked.
I will try to set the datatype to float32 and see if it improves performance.

Reply:
@hermancollin I changed the datatype to float32 but I haven't tested its impact on a large image.